step-capstone/src/styles/App.css

```diff
@@ -0,0 +1,9 @@
+.app {
```

Comment:
I think the map would be rendered in the Trip component since that's the interface for designing individual trips whereas the App component is kind of the user's dashboard.

Reply:
ok i agree. just had this for now so I could see it filling 100% of screen. should be easy to add a style for trip and map will just switch to 100% of that.